Form does not ‘transfer ownership’.

**Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!**


**Describe the bug**
A form does not ‘transfer ownership’. 

Clicking the red ‘Transfer ownership’ button in the form settings and the subsequent process in the pop-up window triggers an error message. The ownership of the form is not transferred. 

**To Reproduce**
Steps to reproduce the behavior:

1. Go to the Settings of a form'
2. Click on red ‘Transfer ownership’ button
3. Fill out the form of the pop-up window: "Search for a user" and "Confirmation text". Click on "I have understood, transfer form"
4. Creates an error: "Errors when transferring ownership"
5. The following can be found in the system log `Fehler	no app in context	
Exception OCA\Forms\Controller\ApiController::updateForm(): Argument #2 ($keyValuePairs) must be of type array, null given, called in /home/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php on line 230 in file '/home/www/nextcloud/apps/forms/lib/Controller/ApiController.php' line 242
Exception thrown: Exception `


**Expected behavior**
Successful transfer of ownership

**Screenshots**
![Image](https://github.com/user-attachments/assets/82bd878c-1b33-4463-b62d-0edd4d0d39aa)

![Image](https://github.com/user-attachments/assets/47b40cb9-f7cb-4964-bb4a-01f7778ffb54)

![Image](https://github.com/user-attachments/assets/60a3a783-3f48-45c4-8e1f-66a3c783a7b6)



**Nextcloud (please complete the following information):**
- Nextcloud 28.0.10 Enterprise
-  PhP 8.2.21
-  Nextcloud-Version: [e.g. 19.0.0]
-  Forms-Version: 4.3.1

**Desktop (please complete the following information):**

-   OS: Linux SuSe (Tumbleweed, same in Win 11
-   Browser Firefox (latest), Same in Chrome (latest)
```
Open your console, reload your page and/or do the action leading to this issue and copy/paste the log in this thread.
```
08:22:29.749 XHRPATCH
muster.de/ocs/v2.php/apps/forms/api/v3/forms/8
[HTTP/1.1 500 Internal Server Error 105ms]

08:22:29.858 [ERROR] forms: Error while transfering form ownership 
Object { app: "forms", uid: "Max Muster", level: 2, error: {…} }
NcSettingsSection-V9DkkSuI-DIvibN42.chunk.mjs:941:434
    log muster.de/apps/forms/js/NcSettingsSection-V9DkkSuI-DIvibN42.chunk.mjs:941
    error muster.de/apps/forms/js/NcSettingsSection-V9DkkSuI-DIvibN42.chunk.mjs:941

...and so on

--- a/src/components/SidebarTabs/TransferOwnership.vue
+++ b/src/components/SidebarTabs/TransferOwnership.vue
@@ -188,7 +188,9 @@ export default {
 							id: this.form.id,
 						}),
 						{
-							ownerId: this.selected.shareWith,
+							keyValuePairs: {
+								ownerId: this.selected.shareWith,
+							},
 						},
 					)
 					showSuccess(

